### ISSUE TITLE
Invalid async chunk reference to `async-burger`

### ISSUE THREAD

**Issue by MomoLangenstein** [thedirtyfew/dash-extensions#188]
When using the `eager_loading=True` option in dash, I came across the following error

```
  File "/home/gitpod/.pyenv/versions/3.8.13/lib/python3.8/site-packages/dash/dash.py", line 800, in _generate_scripts_html
    + self._collect_and_register_resources(
  File "/home/gitpod/.pyenv/versions/3.8.13/lib/python3.8/site-packages/dash/dash.py", line 742, in _collect_and_register_resources
    _relative_url_path(
  File "/home/gitpod/.pyenv/versions/3.8.13/lib/python3.8/site-packages/dash/dash.py", line 721, in _relative_url_path
    modified = int(os.stat(module_path).st_mtime)
FileNotFoundError: [Errno 2] No such file or directory: '/home/gitpod/.pyenv/versions/3.8.13/lib/python3.8/site-packages/dash_extensions/async-burger.js'
```

After a bit of looking online it seems that this file would have been part of chunked loading of components (see https://community.plotly.com/t/dash-how-to-implement-chunked-loading-of-custom-components/55366), specifically the `burger` component which was removed in v0.1.0:

https://github.com/thedirtyfew/dash-extensions/blob/87a9c9ce4ff9731e9f8fb27a40d6ec1b501b6c0a/CHANGELOG.md?plain=1#L33-L57

It seems that some references to this component remain:

https://github.com/thedirtyfew/dash-extensions/blob/87a9c9ce4ff9731e9f8fb27a40d6ec1b501b6c0a/package.json#L29

https://github.com/thedirtyfew/dash-extensions/blob/e43cb965766e2b846892402938d7f464b3691445/MANIFEST.in#L3-L4

and most notably

https://github.com/thedirtyfew/dash-extensions/blob/87a9c9ce4ff9731e9f8fb27a40d6ec1b501b6c0a/dash_extensions/__init__.py#L34-L65

which is used to generate the eager loading script imports.

Would it be possible to remove `"burger"` from this list (and other possibly spurious references) to fix the eager loading? Thank you in advance for your help :)

**Comment by emilhe** [comment]
@MomoLangenstein The issue should be resolved in version `0.1.4`. Please let me know, if you still see any issue(s).

**Comment by MomoLangenstein** [comment]
> @MomoLangenstein The issue should be resolved in version `0.1.4`. Please let me know, if you still see any issue(s).

It now works! Thanks for the quick fix and release!
